ha/alert/pcs/alert_generator.py

```diff
@@ -120,10 +120,23 @@ def send_resource_iem(self):
         """
         Send Resource level IEM to user
         """
-        pass
+        task = self._crm_env['CRM_alert_task']
+        resource = self._crm_env['CRM_alert_rsc']
+        node = self._crm_env['CRM_alert_node']
+        desc = self._crm_env['CRM_alert_desc']
+        rc = self._crm_env['CRM_alert_target_rc']
+        Log.info(f"Resource operation {task} for {resource} on {node}: Desc: {desc}, rc: {rc}")
+        res_iems = Conf.get(const.IEM_INDEX, "resource")
+        if resource in res_iems["resources"]:
+            for module in res_iems["modules"]:
+                if task == res_iems["modules"][module]["SearchKey"]:
+                    msg = Template(res_iems["modules"][module]["IEM"]).substitute(host=node)
+                    Log.info(f"{msg}")
+                    syslog.syslog(msg)
 
     def send_fencing_iem(self):
```

Comment:
Why do we need this function ?

Reply:
It will not generate any iem but it will log fencing action cause by which node and which node got stonith.